internal/topo/topo.go

```diff
@@ -154,7 +154,7 @@ func (s *Topo) Open() <-chan error {
 		defer s.mu.Unlock()
 		var err error
 		if s.store, err = state.CreateStore(s.name, s.qos); err != nil {
-			fmt.Println(err)
```

Comment:
Why changing this?

Reply:
this can not print the error info context , do not know which rule have what error